src/DbBackupsTracker.cpp

```diff
@@ -162,8 +162,7 @@ bool DbBackupsTracker::isBackupRequired() const
         int backupCCN = tryGetCredentialsDbBackupChangeNumber();
         int backupDCN = tryGetDataDbBackupChangeNumber();
 
-        return (backupCCN < credentialsDbChangeNumber || backupDCN < dataDbChangeNumber);
-
+        return isDbBackupChangeNumberLower(backupCCN, backupDCN);
```

Comment:
this will fail when the monitored file has 0xFF and the number on device wraps over to 0x00.
why not !isDbBackupChangeNumberGreater?

Reply:
Fixed in 6b9241d